desktop/libs/azure/src/azure/conf.py

```diff
@@ -75,7 +75,12 @@ def get_default_abfs_url():
   return ABFS_CLUSTERS['default'].WEBHDFS_URL.get()
 
 def get_default_abfs_fs():
-  return ABFS_CLUSTERS['default'].FS_DEFAULTFS.get()
+  default_fs = core_site.get_raz_default_abfs_fs()
+  
+  if core_site.get_raz_api_url() and default_fs:
```

Comment:
Since it can exist without raz, then there is no need to check for `core_site.get_raz_api_url()` here @romainr 
Removing it

Reply:
And also correcting the naming